_translator-files/po/fr/Include-Server-Commands.po

```diff
@@ -30,17 +30,15 @@ msgstr "`-e` ou `--directoryserver`  Enregistrer le serveur sur un annuaire (par
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:4
-#, fuzzy, no-wrap
-#| msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Directories) for further information. \n"
+#, no-wrap
 msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Directories) for further information.\n"
-msgstr "`--directoryfile` Se souvenir des serveurs enregistrés même si l'annuaire est redémarré. Annuaires seulement. Voir [ce guide](Directories) pour plus d'informations. \n"
+msgstr "`--directoryfile` Se souvenir des serveurs enregistrés même si l'annuaire est redémarré. Annuaires seulement. Voir [ce guide](Directories) pour plus d'informations.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:5
-#, fuzzy, no-wrap
-#| msgid "`-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only. \n"
+#, no-wrap
 msgid "`-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only.\n"
-msgstr "`-f` ou `--listfilter`     Liste blanche des serveurs s'enregistrant sur la liste des serveurs, format `ip address 1[;ip address 2]` Annuaires uniquement. \n"
+msgstr "`-f` ou `--listfilter`     Liste blanche des serveurs s'enregistrant sur la liste des serveurs, format `ip address 1[;ip address 2]` Annuaires uniquement.\n"
```

Comment:
Shouldn't this have the extra space before `ou`?

Reply:
Good catch. I added it.